tests/data/controlbox_controlbox.json

```diff
@@ -0,0 +1,142 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.17095.2",
+                    "sysDescr": "TH-332B",
+                    "sysContact": "<private>",
+                    "version": null,
+                    "hardware": null,
```

Comment:
You could add this data, seems its just the sysDescr

Reply:
You are talking about "version" and "hardware" right?
Because I can't find them. Snmpwalk result: https://pastebin.com/2Y1grgjN

Reply:
The sysdescr contains the hardware?

Reply:
Yes, the hardware is same as sysdescr.

Reply:
You can add it with
`sysDescr_regex: '/(?<hardware>.*)/'`

https://docs.librenms.org/Developing/os/Initial-Detection/#yaml-based-os-discovery

Reply:
With `sysDescr_regex: '/(?<hardware>.*)/'` , device is discovering as unknown. If it's possible to keep it without `sysDescr_regex: '/(?<hardware>.*)/'`